extensions/skins/conversation_v1/player.html

```diff
@@ -2,107 +2,258 @@
+          </span>
+        </div>
 
-            <table class="conversation-skin-card-row-container">
-              <tr class="conversation-skin-card-row">
-                <td class="conversation-skin-row-avatar" valign="top">
-                  <img class="conversation-skin-row-avatar-img" src="/images/general/oppia_black_48px.png">
-                </td>
-                <td class="conversation-skin-oppia-output conversation-skin-oppia-content">
-                  <div class="protractor-test-conversation-content" angular-html-bind="state.content"></div>
-                </td>
-              </tr>
-            </table>
-
-            <div ng-repeat="responsePair in state.answerFeedbackPairs track by $index">
-              <table class="conversation-skin-card-row-container">
-                <tr ng-show="responsePair.learnerAnswer" class="conversation-skin-card-row">
-                  <td class="conversation-skin-row-avatar" valign="top">
-                    <img class="conversation-skin-row-avatar-img img-circle" ng-src="<[profilePicture]>">
-                  </td>
-                  <td class="conversation-skin-learner-answer" angular-html-bind="responsePair.learnerAnswer"></td>
-                </tr>
-              </table>
-              <table ng-show="responsePair.oppiaFeedback" class="conversation-skin-card-row-container">
-                <tr class="conversation-skin-card-row">
-                  <td class="conversation-skin-row-avatar" valign="top">
-                    <img class="conversation-skin-row-avatar-img" src="/images/general/oppia_black_48px.png">
-                  </td>
-                  <td class="conversation-skin-oppia-output conversation-skin-oppia-feedback protractor-test-conversation-feedback" angular-html-bind="responsePair.oppiaFeedback"></td>
-                </tr>
-              </table>
+        <div class="conversation-skin-animate-card-contents" ng-class="{'animate-card-change': startCardChangeAnimation}">
+          <div class="conversation-skin-tutor-card-top-section">
+            <img class="conversation-skin-initial-oppia-avatar" src="/images/general/o_black_72px.png">
+            <div class="protractor-test-conversation-content" angular-html-bind="activeCard.content">
             </div>
+          </div>
+
+          <div ng-if="activeCard.answerFeedbackPairs.length > 1">
+            <h4 class="conversation-skin-responses-dropdown-container" ng-click="toggleShowPreviousResponses()">
+              <span class="conversation-skin-responses-dropdown-text">
+                Previous responses
+                <span ng-if="!arePreviousResponsesShown">
+                  <span class="glyphicon glyphicon-play" style="font-size:10px; margin-bottom: -4px;"></span>
+                </span>
+                <span ng-if="arePreviousResponsesShown">
+                  <span class="glyphicon glyphicon-triangle-bottom" style="font-size:10px; margin-bottom: -4px;"></span>
```

Comment:
Ditto. This style is also duplicated with the one on line 52. Consider moving it into a class.